Tests/Test.Eyes.Selenium.DotNet/ApiTests/TestExceptions.cs

```diff
@@ -34,13 +38,9 @@ public void TestEyesExceptions()
                 conf.SetTestName("test");
                 eyes.SetConfiguration(conf);
                 eyes.Open(driver);
-
-                TestResults results = eyes.Close(false);
-                results?.Delete();
```

Comment:
change `appIdOrName` to `appName` and `scenarioIdOrName` to `testName` and merge tests.